common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryTokenResponse.java

```diff
@@ -85,4 +101,58 @@ public String getIdToken() {
     public void setIdToken(String idToken) {
```

Comment:
Unrelated to this PR... just didn't catch this on the last one.  We should consider moving IDToken to base TokenResponse... http://openid.net/specs/openid-connect-core-1_0.html#TokenResponse

Reply:
Roger that 👍

Reply:
Resolved in 9110b3a84e222b8463e7fb8b93ecf42771423acc